prepare

```diff
@@ -41,28 +41,6 @@ pull_snap() {
         print_error "Could not set up Snap! git submodule.\nPlease make sure that you have access to the Internet and git is installed in your system."
         exit 1
     fi
-    cd src/snap
-    git checkout master
-    if ! git pull; then
-        print_error "Could not pull latest Snap! version from Git.\nPlease make sure that you have access to the Internet and git is installed in your system."
-        cd ../..
-        exit 1
-    fi
```

Comment:
Closing this PR.
I'll send a new PR to fix the new repo break.

Waiting to know the solution chosen (pointing to jmoenig:master again or pointing to a fork with the last changes)